enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -173,27 +184,70 @@ Usage of the Thanos querier component inside cluster monitoring has many advanta
 
 See [5] and [6] for more details.
 
+#### Alerts/Recording Rules Aggregation
+
+Thanos Querier only solves aggregating the query path. For aggregating recording rules as well as alerting rules
+another component Thanos Ruler is deployed. This component is responsible for hosting user defined recording rules
+as well as alerting rules.
+
+Thanos Ruler is deployed via the user workload monitoring prometheus operator via a new CRD called ThanosRule.
+The same _deny list_ setting is applied to user workload monitoring prometheus operator such that the user cannot
+deploy its own Thanos Ruler instances in user namespaces.
+
+Thanos Ruler queries Thanos Querier to resolve recording and alerting rules.
+This does have implications on the availability of user defined alerts.
+In order for alerts to work the following systems need to be up:
+
+1. Thanos Ruler
+2. Thanos Querier
+3. User workload Prometheus
+4. Cluster monitoring Prometheus
+
+If there is no need to aggregate user defined metrics and cluster level metrics in user alerts,
+higher availability can be achieved by hosting user defined alerting rules on the
+user workload Prometheus instances.
+
+This is enabled by setting a label on the PrometheusRule custom resource which is deployed by the user.
+Prometheus Operator then dispatches the deployment of this rule on the user workload monitoring Prometheus instance
+rather than on the Thanos Ruler instance.
+In order for alerts to work in this case the following systems need to be up:
+
+1. User workload Prometheus
+
 ### Tenancy
 
+#### Queries
+
 ![](user-monitoring-request.png)
 
 Tenancy is achieved by leveraging the existing topology that is protecting cluster monitoring Prometheus already today.
 The kube-rbac-proxy sidecar will be deployed along with prom-label-proxy in the Thanos Querier Deployment.
 
 For details about the tenancy model see the README for prom-label-proxy [2].
 
+#### Rules and Alerting tenancy
+
+![](user-monitoring-rules-request.png)
+
+OpenShift consoles executes queries against the `/rules` and `/alerts` endpoint of Prometheus and Alertmanager
```

Comment:
What about Thanos Ruler `/rules`?

Reply:
@bwplotka indeed, I will add this here :+1:

Reply:
to clarify the `/rules` endpoint of Thanos Ruler adheres to the same API as Prometheus `/rules` correct?

Reply:
I separated out prometheus/thanos-ruler and alertmanager in separate paragraphs.